docs/2.10.0/docs/app-dev/bindings-java/quickstart.rst

```diff
@@ -580,21 +580,21 @@ It consists of the application in file ``IouMain.java``. It uses the class ``Iou
 
    .. literalinclude:: quickstart/template-root/src/main/java/com/daml/quickstart/iou/IouMain.java
       :language: java
-      :lines: 46-49
+      :lines: 45-48
```

Comment:
Can you explain ?

Reply:
Line numbers are off-by-one because one import (line) got removed at the top of the file.